develop/dev-guide-get-data-from-single-table.md

```diff
@@ -133,11 +134,11 @@ SELECT * FROM authors WHERE birth_year = 1998;
 </div>
 <div label="Java" href="filter-java">
 
-For Java programs, we want to use the same SQL to handle data query request with dynamic parameters.
+In Java, you can use the same SQL to handle data query request with dynamic parameters.
 
-Splicing parameters into a SQL statement may be a method, but it is not a good idea, as it poses a potential [SQL Injection](https://en.wikipedia.org/wiki/SQL_injection) risk to our application.
+This can be done by splicing parameters into a SQL statement. However, this method will pose a potential [SQL Injection](https://en.wikipedia.org/wiki/SQL_injection) risk to the security of the  application.
```

Comment:
```suggestion
This can be done by concatenating parameters into a SQL statement. However, this method will pose a potential [SQL Injection](https://en.wikipedia.org/wiki/SQL_injection) risk to the security of the application.
```

Reply:
@Icemap Please help confirm this change.